gr-utils/python/modtool/templates/gr-newmod/python/build_utils_codes.py

```diff
@@ -19,30 +18,29 @@
 # the Free Software Foundation, Inc., 51 Franklin Street,
 # Boston, MA 02110-1301, USA.
 #
+from __future__ import unicode_literals
```

Comment:
is this necessary or are we still trying to get rid of all the future imports?

Reply:
I did not remove future imports from the newmoddir. Shall I do it?